library/core/src/mem/maybe_uninit.rs

```diff
@@ -851,4 +851,121 @@ impl<T> MaybeUninit<T> {
     pub fn slice_as_mut_ptr(this: &mut [MaybeUninit<T>]) -> *mut T {
         this.as_mut_ptr() as *mut T
     }
+
+    /// Copies the elements from `src` to `this`, returning a mutable reference to the now initalized contents of `this`.
+    ///
+    /// If `T` does not implement `Copy`, use [`clone_from_slice`]
+    ///
+    /// This is equivalent to [`slice::copy_from_slice`].
```

Comment:
It's good to mention the relationship of these functions with slice::copy_from_slice/clone_from_slice, but 'equivalent' wouldn't be entirely correct. Is there a better way to describe the similarity to those functions?

Reply:
Its similarity, so why not "This is similar to [`slice::copy_from_slice`]." ?

Reply:
Yes, it's nicer !

Reply:
changed it to similar